tests/plugins/font/manual/fontcommand.md

```diff
@@ -0,0 +1,30 @@
+@bender-tags: 4.14.0, feature, 3306, 3728
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, font, enterkey, elementspath, notification
```

Comment:
It would be nice to also add sourcearea plugin, so we can see if modes work correctly.